account/src/main/java/bisq/account/protocol_type/LoanProtocolType.java

```diff
@@ -20,15 +20,18 @@
 import bisq.common.util.ProtobufUtils;
 
 public enum LoanProtocolType implements ProtocolType {
+    UNSPECIFIED,
     COLLATERALIZED,
     REPUTATION;
 
+    private static final String protobufPrefix = "LOANPROTOCOLTYPE_";
+
     @Override
     public bisq.account.protobuf.LoanProtocolType toProto() {
-        return bisq.account.protobuf.LoanProtocolType.valueOf(name());
+        return bisq.account.protobuf.LoanProtocolType.valueOf(protobufPrefix + name());
     }
 
     public static LoanProtocolType fromProto(bisq.account.protobuf.LoanProtocolType proto) {
-        return ProtobufUtils.enumFromProto(LoanProtocolType.class, proto.name());
+        return ProtobufUtils.enumFromProto(LoanProtocolType.class, proto.name().replace(protobufPrefix, ""));
```

Comment:
Do name conversion generically in ProtobufUtils (see comment there)

Reply:
Done